bootstrap.bat

```diff
@@ -196,6 +196,29 @@ del sikuli*.jar
 @echo on
 
 
+@REM Installing OBS
+
+IF "%APPVEYOR%"=="True" GOTO SkipOBS
+
+IF NOT EXIST "C:\Program Files (x86)\obs-studio\bin\32bit\obs32.exe" (
```

Comment:
What if they have 32-bit windows? Should check for both `Program Files` and `Program Files (x86)`